Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -15857,6 +15857,14 @@ void WebPageProxy::setPresentingApplicationAuditToken(const audit_token_t& prese
 }
 #endif
 
+#if ENABLE(CONTENT_EXTENSIONS)
+void WebPageProxy::shouldOffloadIFrameForHost(const String& host, CompletionHandler<void(bool)>&& completionHandler)
+{
+    auto granted = protectedWebsiteDataStore()->resourceMonitorThrottler().tryAccess(host);
```

Comment:
`auto` -> `bool`. Also, boolean variables need a prefix in WebKit as per coding style. Here I would call it `wasGranted`.